deploy/upstream/values.yaml

```diff
@@ -7,21 +7,21 @@ writeStatusName: '""'
 olm:
   replicaCount: 1
   image:
-    ref: quay.io/coreos/olm@sha256:995a181839f301585a0e115c083619b6d73812c58a8444d7b13b8e407010325f
+    ref: quay.io/openshift/origin-operator-lifecycle-manager:latest
```

Comment:
We probably want Alpine images for upstream.

Reply:
do we have any built?